src/SigasiProjectCreator/ConverterHelper.py

```diff
@@ -132,6 +132,12 @@ def parse_and_create_project(usage, parse_file):
 
             sigasi_project_file_creator.add_link(relative_file_path, relative_location_path)
 
+    # For the time being, we assume that absolute paths are used here (e.g. in a simulator install tree)
+    # TODO support relative paths (should be part of a more general path handling overhaul?)
```

Comment:
isn't this as simple as uvm_location = uvm_location.absolute()` if we're using pathlib (or `os.path.abspath` if we're not)? I see no reason to postpone this to later overhauls.